.env.template

```diff
@@ -7,8 +7,8 @@ JWT_TOKEN=
 PROJECT=datahub
 # base domain to which prefixes are added such as STAGE and specific services
 DOMAIN_BASE=datahub.io
-# stage e.g. testing, staging, production
-STAGE=staging
+# stage e.g. testing, production
+STAGE=production
```

Comment:
Probably `testing` is better as a default

Reply:
OK I will keep it in mind for the future.